includes/admin/customers/class-customer-addresses-table.php

```diff
@@ -66,9 +66,10 @@ public function column_default( $item, $column_name ) {
 		switch ( $column_name ) {
 
 			case 'type' :
-				$value = ( 'primary' === $item['type'] )
-					? esc_html_e( 'Primary',   'easy-digital-downloads' )
-					: esc_html_e( 'Secondary', 'easy-digital-downloads' );
+				$value = ucwords( edd_get_address_type_label( $item['type'] ) );
```

Comment:
This has just come up on a support ticket--not all languages handle capitalization the same way (German is a specific example). We should not be forcing upper/lower case on translatable strings.